code/modules/antagonists/clockcult/clockcult.dm

```diff
@@ -193,6 +193,17 @@
 	var/list/objective
 	var/datum/mind/eminence
 
+/datum/team/clockcult/New(starting_members)
+	. = ..()
+	START_PROCESSING(SSobj,src)
```

Comment:
Why don't we have a gamemode subsystem or something

Reply:
Mostly because only things that could use is this and rev headcount maybe.

Reply:
also SSticker is the gamemode subsystem